### PR TITLE
feat(blade): add aria hidden to icons

### PR DESCRIPTION
**Pull request by anuraghazra**
- Added support for accessibilityHidden in mapA11yProps.
- Added aria hidden properties in Svg.

### CODE REVIEW

File: packages/blade/src/utils/makeAccessible/makeAccessible.native.ts [hunk 1]
```diff
@@ -37,6 +37,18 @@ const makeAccessible = (props: Partial<AccessibilityProps>): Record<string, unkn
       continue;
     }
 
+    // handle accessibilityHidden
+    // in react-native we have to platform specific values for it.
+    if (propKey === 'hidden') {
+      if (propValue === true) {
+        newProps.accessibilityElementsHidden = true;
+        newProps.importantForAccessibility = 'no-hide-descendants';
+      }
```
Comment (kamleshchandnani) [inline packages/blade/src/utils/makeAccessible/makeAccessible.native.ts:46]: shouldn't these props be part of the map and if it's present you directly use that value?

also, why are we deleting  `accessibilityHidden` if propKey === 'hidden'?

Comment (anuraghazra) [reply packages/blade/src/utils/makeAccessible/makeAccessible.native.ts:46]: > shouldn't these props be part of the map and if it's present you directly use that value?
This is special case, no way to put it in the map

We don't need `accessibilityHidden` to be passed in the components thus we delete it, we only need to passthrough accessibilityElementsHidden, importantForAccessibility

File: packages/blade/src/utils/makeAccessible/makeAccessible.native.ts [hunk 2]
```diff
@@ -37,6 +37,18 @@ const makeAccessible = (props: Partial<AccessibilityProps>): Record<string, unkn
       continue;
     }
 
+    // handle accessibilityHidden
+    // in react-native we have to platform specific values for it.
```
Comment (kamleshchandnani) [inline packages/blade/src/utils/makeAccessible/makeAccessible.native.ts:41]:
```suggestion
    // in react-native we have 2 platform-specific values for it.
```

File: packages/blade/src/utils/makeAccessible/makeAccessible.native.ts (hunk 1 above)
Comment (kamleshchandnani) [reply packages/blade/src/utils/makeAccessible/makeAccessible.native.ts:46]: is `accessibilityHidden` is just the key to enable these two props and hence we are deleting it?

Comment (anuraghazra) [reply packages/blade/src/utils/makeAccessible/makeAccessible.native.ts:46]: Yes.

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**